publications.md

```diff
@@ -20,31 +20,31 @@ Regarding the language design, there are two main references:
 
 > **"Effects as Capabilities: Effect Handlers and Lightweight Effect Polymorphisms"**.\
 > Jonathan Immanuel Brachthäuser, Philipp Schuster, and Klaus Ostermann. 2020.\
-> In _Proc. ACM Program. Lang._, 4 (OOPSLA), Article 126 ([ACM](https://dl.acm.org/doi/10.1145/3428194), [Extended Techreport](https://se.cs.uni-tuebingen.de/publications/brachthaeuser20effekt/)).
+> In _Proc. ACM Program. Lang._, 4 (OOPSLA), Article 126 ([ACM](https://doi.org/10.1145/3428194), [Extended Techreport](https://se.cs.uni-tuebingen.de/publications/brachthaeuser20effekt/)).
 
 
 > **"Effects, Capabilities, and Boxes: from scope-based reasoning to type-based reasoning and back"**.\
 > Jonathan Immanuel Brachthäuser, Philipp Schuster, Edward Lee, and Aleksander Boruch-Gruszecki.\
-> In _Proc. ACM Program. Lang._, 6 (OOPSLA), Article 76 ([ACM](https://dl.acm.org/doi/abs/10.1145/3527320), [Extended Techreport](https://se.cs.uni-tuebingen.de/publications/brachthaeuser22effects/))
+> In _Proc. ACM Program. Lang._, 6 (OOPSLA), Article 76 ([ACM](https://doi.org/10.1145/3527320), [Extended Techreport](https://se.cs.uni-tuebingen.de/publications/brachthaeuser22effects/))
 
 
 ### Compilation of Effect Handlers
 
 > **"From Capabilities to Regions: Enabling Efficient Compilation of Lexical Effect Handlers"**.\
 > Marius Müller, Philipp Schuster, Jonathan Lindegaard Starup, Klaus Ostermann, and Jonathan Immanuel Brachthäuser. 2023.
-> In _Proc. ACM Program. Lang._, 7 (OOPSLA2), Article 225. ([Techreport](https://se.cs.uni-tuebingen.de/publications/mueller23lift/), [ACM](http://dx.doi.org/10.1145/3622831))
+> In _Proc. ACM Program. Lang._, 7 (OOPSLA2), Article 225. ([Techreport](https://se.cs.uni-tuebingen.de/publications/mueller23lift/), [ACM](https://dx.doi.org/10.1145/3622831))
```

Comment:
Why the `dx.` prefix? 🤔 
As far as I can tell, it's deprecated (https://www.crossref.org/display-guidelines/, section Why not use `dx` as in `http://dx.doi.org/`?)
```suggestion
> In _Proc. ACM Program. Lang._, 7 (OOPSLA2), Article 225. ([Techreport](https://se.cs.uni-tuebingen.de/publications/mueller23lift/), [ACM](https://doi.org/10.1145/3622831))
```